CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Fixed a bad type conversion which could cause underflow on a window resize
 - Alacritty now spawns a login shell on macOS, as with Terminal.app and iTerm2
+- Fixed the problem of zombie processes by detaching the child process.
```

Comment:
No punctuation should be at the end of change log comments.

I'd also like a mention to the feature in question here (url-launching). Without that it's not really clear under which circumstances these zombies were created.